torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -428,8 +428,9 @@ struct KinetoThreadLocalState : public ProfilerStateBase {
             [this](ExtraFields<EventType::TorchOp>& i) { invokeCallback(i); },
             [this](ExtraFields<EventType::Backend>& i) { invokeCallback(i); },
             [](auto&) {}));
-
-        kineto_events_.emplace_back(e, config_.experimental_config.verbose);
+        bool enable_python_stack =
+            config_.experimental_config.verbose || config_.with_stack;
+        kineto_events_.emplace_back(e, enable_python_stack);
```

Comment:
This will likely cause a regression, as this will turn on the older version of stack tracing

Reply:
Is there a newer version that is supposed to be used? I was trying to turn this back on because all of the tutorials/documentation suggest that this should work.

Reply:
Right now using with_stack only turns on tracing for jit